libs/ui/src/lib/header/header.module.css

```diff
@@ -1,12 +1,12 @@
 .header {
-  background: '#2d3741';
+  background: #2d3741;
```

Comment:
Bruke token fra DS for bakgrunnsfargen her?

Reply:
Tenker du kun i dette tilfellet eller at vi generelt bør begynne å bruke token fra ds?

Reply:
Generelt. Tror kanskje vi landet på at vi bruker Digdirs token og ikke egen (?) Men bør avklares med @matfuru

Reply:
Ja, vi skulle ha begynt med det. Jeg har nå sjekket de siste committene ser ikke ut som det er brukt noen steder. Men vi kan ta det opp med resten av teamet og begynne å bruke det, både farger og størrelser. 👍

Reply:
jeg har fjernet alt gammelt..når er det designsystemet som gjelder over alt:)